### PR TITLE
Added ability to retrieve current clients target topics

### PR DESCRIPTION
**Pull request by UnaffiliatedCode**
Currently I am looking at multiple dynamic deployments of this library and I'm looking to implement checks prior to initialized polling which need to verify that the necessary topics have been assigned to the consumer client. 

Please advise if you require additional unit tests for the ConsumerGroup  path.

### CODE REVIEW

File: pkg/kgo/consumer.go [hunk 1]
```diff
@@ -758,6 +758,32 @@ func (cl *Client) AddConsumeTopics(topics ...string) {
 	cl.triggerUpdateMetadataNow("from AddConsumeTopics")
 }
 
+// GetConsumeTopics retrives a list of current topics to be consumed.
+//
+// Note that If you are directly consuming and specified ConsumePartitions,
+// this function will not be aware of the consumption topic target(s).
```
Comment (twmb) [inline pkg/kgo/consumer.go:764]: wdym this isn't aware of partition consuming. I think looking at the code, `tps` is added to for partition consuming as well.

(also If => if)

Comment (UnaffiliatedCode) [reply pkg/kgo/consumer.go:764]: Upon review I believe you to be correct. I'll amend the commit.
(I'll also fix the if XD )

Comment (UnaffiliatedCode) [reply pkg/kgo/consumer.go]: This has been completed.

**Review by twmb — APPROVED** [review]

**Comment by twmb** [comment]
Sorry for the delay, the last few months were much busier than anticipated. Releasing imminently!

**Comment by UnaffiliatedCode** [comment]
> Sorry for the delay, the last few months were much busier than anticipated. Releasing imminently!

All good. Thanks for the merge

**Pull request merged**